man/machinery-inspect-docker.1.md

```diff
@@ -0,0 +1,111 @@
+
+## inspect — Inspect Running System
+
+### SYNOPSIS
+
+`machinery inspect-container` [OPTIONS] IMAGENAME / IMAGEID
+
+`machinery` help inspect-container
+
+
+### DESCRIPTION
+
+The `inspect-container` command inspects a container image by creating, starting and inspecting a container from the provided image
+and generates a system description from the gathered data. After the inspection the container will be killed and removed again.
+This approach ensures that no containers and images are affected by the inspection.
+
+The system data is structured into scopes, controlled by the
+`--scope` option.
+
+**Note**:
+Machinery will always inspect all specified scopes, and skip scopes which
+trigger errors.
+
+
+### ARGUMENTS
+
+  * `IMAGENAME / IMAGEID` (required):
+    The name or id of the image to be inspected. The name / id will also be
+    used as the name of the stored system description unless another name is
+    provided with the `--name` option.
+
+
+### OPTIONS
+
+  * `-d`, `--docker` (optional):
+    Defines the type of the image to be inspected as docker image.
```

Comment:
If I remember correctly the `-d` switch doesn't influence the inspection, yet. I'd say that as only docker is supported for now it's the default.